Prox/Prox/Utilities/AppConstants.swift

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+public enum AppBuildChannel {
+    case Developer
+    case Enterprise
+    case Release
+}
+
+
+
+public struct AppConstants {
+
+    public static let IsRunningTest = NSClassFromString("XCTestCase") != nil
+
+    /// Build Channel.
+    public static let BuildChannel: AppBuildChannel = {
+        #if MOZ_CHANNEL_ENTERPRISE
+            return AppBuildChannel.Enterprise
+        #elseif MOZ_CHANNEL_RELEASE
+            return AppBuildChannel.Release
+        #else
+            return AppBuildChannel.Developer
+        #endif
+    }()
+
+    /// Flag indiciating if we are running in Debug mode or not.
+    public static let isDebug: Bool = {
+        #if MOZ_CHANNEL_DEBUG
```

Comment:
nit: I feel like we should re-use `BuildChannel` here and below, e.g. `return BuildChannel == .Developer`:
* It re-uses existing code that we (hopefully) know works, which means it can be slightly less error-prone.
* If the configuration changes (e.g. we remove the developer channel), we're more likely to catch that error in `isDebug`.